features/0453-issue-credential-v2/README.md

```diff
@@ -147,7 +147,7 @@ Description of attributes:
 Credential Format | Format Value | Link to Attachment Format | Comment |
 --- | --- | --- | --- | 
 DIF Credential Manifest | `dif/credential-manifest@v1.0` | [`propose-credential` attachment format](../0511-dif-cred-manifest-attach/README.md#propose-credential-attachment-format) | 
-Hyperledger Indy Credential Abstract | `hlindy/cred-abstract@v2.0` | [`cred abstract` format](../0592-indy-attachments/README.md#cred-abstract-format)|
+Hyperledger Indy Credential Abstract | `hlindy@cred-filter@v2.0` | [`cred filter` format](../0592-indy-attachments/README.md#cred-filter-format)|
```

Comment:
I think this identifier should have a / instead of a @ in the first slot: `hlindy/cred-filter@v2.0`

Reply:
oh, you're right

Reply:
Fixed in PR 618.